View/View.php

```diff
@@ -132,6 +133,21 @@ public function setData($data)
     }
 
     /**
+     * Set template variable
+     *
+     * @param $variableName
+     * @param $data
+     *
+     * @return View
+     */
+    public function setTemplateData($variableName, $data)
```

Comment:
What about this situation?
```
$view = $this->view($products, 200)
    ->setTemplate("SiteViewBundle:Category:show.html.twig")
    ->setTemplateVar('products')
    ->setTemplateData('category', $category);
/* more code */
$view->setTemplateData('other_template_data', $otherTemplateData)
```
I think it`s not necessary to change setTemplateData function.